src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -113,6 +130,14 @@
     private Duration reconnectionDelay = new Duration(DEFAULT_RECONNECTION_DELAY);
 
     /**
+     * Time period to wait before attempting to reconnect to primary server
+     * when multiple servers are specified. Only happens when the appender
+     * is currently connected to a secondary.
+     * Default is {@value #DEFAULT_RECONNECTION_PRIMARY_DELAY} milliseconds.
+     */
+    private Duration reconnectionToPrimaryDelay = new Duration(DEFAULT_RECONNECTION_PRIMARY_DELAY);
```

Comment:
The naming of this is really similar to `reconnectionDelay`.  Is there a name that better represents refreshing/reestablishing an existing connection that can be used here?

Perhaps 'reattemptPrimaryConnectionDelay'.  Something else?  I don't know, just brainstorming.  It's minor.